common/src/domain.ts

```diff
@@ -404,8 +404,9 @@ export function newNote(
     height: number = 5,
     shape: NoteShape = "square",
     z: number = 0,
+    fontSize: number = 1,
```

Comment:
This seems unnecessary, as the parameter is not used, right?

Reply:
it is used while creating a new note through PUT request, it gives stable fontSize even if we have multiple lines inside the note, which helps the UI look good, any suggestion on this bro

Reply:
Well it doesn't actually guarantee that. The fontSize number is actually used as a _factor_ for auto-sizing, instead of fixing the size. It may not be the best of designs but that's how it works. So practically a missing/undefined fontSize behaves similarly to size 1.

Reply:
> Well it doesn't actually guarantee that. The fontSize number is actually used as a _factor_ for auto-sizing, instead of fixing the size. It may not be the best of designs but that's how it works. So practically a missing/undefined fontSize behaves similarly to size 1.

ok bro, will test along with others and reach you